src/constants.py

```diff
@@ -32,7 +32,7 @@
 BUFFERS_VALUE_USD_THRESHOLD = 400000
 
 # threshold parameter to generate an alert when receiving kickbacks
-KICKBACKS_ALERT_THRESHOLD = 0.5
+KICKBACKS_ALERT_THRESHOLD = 0.2
```

Comment:
Most of those alerts would not actionable alerts, right?
If you want to monitor for potentially interesting settlements, please use a Dune [like this one](https://dune.com/queries/3668122)